### PR TITLE
Develop3

### PR DESCRIPTION
**Pull request by arunmortha**

### CODE REVIEW

File: .idea/java [hunk 1]
```diff
@@ -1,3 +1,3 @@
 java fullstack
+jdbc and hibernate
```
Comment (Tirumala12345) [inline .idea/java:2]: Pls add  comments

Comment (arunmortha) [reply .idea/java:2]: okay i will check

Comment (Tirumala12345) [reply .idea/java:2]: Also use proper Naming Conventions

Comment (arunmortha) [reply .idea/java:2]: okay sure

**Review by Tirumala12345 — APPROVED** [review]

**Pull request merged**